libraries/SITL/SIM_Temperature_TSYS01.cpp

```diff
@@ -1,5 +1,7 @@
 #include "SIM_Temperature_TSYS01.h"
 
+#if AP_SIM_TEMPERATURE_TSYS01_ENABLED
```

Comment:
Our pattern is to include `SIM_config.h` and then check this, including the .h corresponding to the .cpp after that check has passed.

Reply:
Fixed in follow up here: https://github.com/ArduPilot/ardupilot/pull/28871